english/order-pizza/order-pizza.md

```diff
@@ -1,3 +1,15 @@
 Your hungry and tired, pizza is the only thing that will cheer you up.
 But you dont know what to order.
-Suddenly a [menu](https://www.pizzahut.com.au/) appears!
\ No newline at end of file
+Suddenly a [menu](https://www.pizzahut.com.au/) appears!
+
+You thought long and hard about fried chicken, or perhaps all you can eat
+mongolian stir fry but in the end the only thing that truly matters to a
+programmer is hot delivery pizza. When it's done you'll hang it from your window
+in a pillowcase to keep it cold and hidden from the horrible people that are
+after you.
+
+Pineapple however makes everything better.
+
+Another option also appears a [menu](http://fongspizza.com/) for Fong's pizza.
+
+Fong's serves delicious Asian Pizza and also offers exceptional [Tiki Drinks](http://liquor.com/slideshows/10-tiki-cocktails-to-kick-off-summer/).
```

Comment:
Interesting, in the past we've always allowed overages when URL's came into play, the 80 or in your case the ever generous 120 character limit, still was never designed to accommodate any sort of URL structure.  I typically follow the drupal coding standards which explicitly allow this.  I'm also wondering how you can have coding standards on something which looks explicitly editorial...